error  in ./node_modules/vue2-perfect-scrollbar/dist/vue2-perfect-scrollbar.css

Module build failed (from ./node_modules/postcss-loader/src/index.js):
Error: Cannot find module 'postcss-import'

Just ran into the same thing on build (during development it worked fine):

```
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/src/index.js):
Error: Cannot find module 'postcss-import'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/breadadams/dev/projectname/node_modules/vue2-perfect-scrollbar/postcss.config.js:2:21)
    at Module._compile (module.js:652:30)
    at requireFromString (/Users/breadadams/dev/projectname/node_modules/require-from-string/index.js:28:4)
    at parseJsFile (/Users/breadadams/dev/projectname/node_modules/postcss-load-config/node_modules/cosmiconfig/dist/loadJs.js:15:15)
    at <anonymous>
    at runLoaders (/Users/breadadams/dev/projectname/node_modules/webpack/lib/NormalModule.js:301:20)
    at /Users/breadadams/dev/projectname/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /Users/breadadams/dev/projectname/node_modules/loader-runner/lib/LoaderRunner.js:230:18
    at context.callback (/Users/breadadams/dev/projectname/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at Promise.resolve.then.then.catch (/Users/breadadams/dev/projectname/node_modules/postcss-loader/src/index.js:208:9)
    at <anonymous>

 @ ./src/main.js 23:0-26
 @ ./node_modules/@vue/cli-service/lib/commands/build/entry-lib.js
```

The reason NPM isn't installing that dependency with the library is because it's marked as a devDependency, the solution would be to move it to `dependencies`.

--- a/package.json
+++ b/package.json
@@ -37,7 +37,6 @@
     "babel-loader": "^8.0.2",
     "chai": "^4.1.2",
     "css-loader": "^1.0.0",
-    "cssnano": "^4.1.3",
     "eslint": "^5.5.0",
     "eslint-config-standard": "^12.0.0",
     "eslint-loader": "^2.1.0",
@@ -55,7 +54,6 @@
     "karma-webpack": "^3.0.5",
     "mocha": "^5.2.0",
     "node-sass": "^4.9.3",
-    "postcss-import": "^12.0.0",
     "rollup": "^0.65.2",
     "rollup-plugin-buble": "^0.19.2",
     "rollup-plugin-commonjs": "^9.1.6",
@@ -74,6 +72,8 @@
     "webpack-dev-server": "^3.1.7"
   },
   "dependencies": {
-    "perfect-scrollbar": "^1.4.0"
+    "cssnano": "^4.1.3",
+    "perfect-scrollbar": "^1.4.0",
+    "postcss-import": "^12.0.0"
   }
 }

